src/components/shared/StackSettings.js

```diff
@@ -42,29 +42,37 @@ class StackSettings extends PureComponent<Props> {
     this.props.changeInvertCallstack(e.currentTarget.checked);
   };
 
+  _renderRadioButton(
+    label: string,
+    implementationFilter: ImplementationFilter
+  ) {
+    return (
+      <label className="stackSettingsFilterLabel">
+        <input
+          type="radio"
+          value={implementationFilter}
+          name="stack-settings-filter"
+          title="Filter stack frames to a type."
+          onChange={this._onImplementationFilterChange}
+          checked={this.props.implementationFilter === implementationFilter}
+        />
+        {label}
+      </label>
+    );
+  }
+
   render() {
-    const {
-      implementationFilter,
-      invertCallstack,
-      hideInvertCallstack,
-    } = this.props;
+    const { invertCallstack, hideInvertCallstack } = this.props;
 
     return (
       <div className="stackSettings">
         <ul className="stackSettingsList">
           <li className="stackSettingsListItem">
-            <label className="stackSettingsLabel">
-              Filter:
-              <select
-                className="stackSettingsSelect"
-                onChange={this._onImplementationFilterChange}
-                value={implementationFilter}
-              >
-                <option value="combined">Combined stacks</option>
-                <option value="js">JS only</option>
-                <option value="cpp">C++ only</option>
-              </select>
-            </label>
+            <div className="stackSettingsFilter">
```

Comment:
just wondering if we need this div wrapper -- is it possible to move part of its style to the `<li>` parent ?